--- a/libpcsxcore/misc.c
+++ b/libpcsxcore/misc.c
@@ -660,10 +660,10 @@ void check_AOT(void)
 
 /* zlib helper routines */

-#if defined(USE_MINIZ)
-#error "need real zlib for gz* funcs, sorry"
-#elif defined(HAVE_ZLIB)
+#if defined(HAVE_ZLIB)
 #include <zlib.h>
+#elif defined(USE_MINIZ)
+#error "need real zlib for gz* funcs, sorry"
 #else
 static void *zlib_open(const char *name, const char *mode)
 {
